src/lib/integration_test_cloud_engine/mina_automation.ml

```diff
@@ -134,18 +135,20 @@ module Network_config = struct
         "All nodes in testnet must have unique names.  Check to make sure you \
          are not using the same node_name more than once" ;
 
-    (* GENERATE ACCOUNTS AND KEYPAIRS *)
-    let keypairs =
-      List.take
-        (* the first keypair is the genesis winner and is assumed to be untimed. Therefore dropping it, and not assigning it to any block producer *)
-        (List.drop
-           (Array.to_list (Lazy.force Key_gen.Sample_keypairs.keypairs))
-           1 )
+    (* ACCOUNTS AND KEYPAIRS *)
+    (* ACCOUNTS AND KEYPAIRS *)
+    let before = Time.now () in
```

Comment:
In the interest of keeping the GCP engine as close to what it was before building the new engine, I wonder if we should leave the key generation as is. I'll raise in the stand-up.